scripts/pull.sh

```
@@ -243,19 +244,21 @@ function pull {
             if [ $merge_code != 0 ] ; then
                 echo
            fi
-        else 
+        elif [ "$choice" == "2" ]; then
            rebase_branch $1 $2 "true" $interactive $interactive
            mode="rebase"
            if [ $rebase_code != 0 ] ; then
```

Shellcheck (0.10.0)

[warning] rebase_code is referenced but not assigned.

(SC2154)